From f9e3b85c9857e7124a1507f71ea220b479532abc Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Thu, 27 Apr 2023 13:17:54 +0800
Subject: [PATCH 19/68] Revert "drm/virtio: Fix error code in
 virtio_gpu_object_shmem_init()"

This reverts commit c5fe3fba1b7bfecb6f17f93a433782b8500fe377.
---
 drivers/gpu/drm/virtio/virtgpu_object.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/virtio/virtgpu_object.c b/drivers/gpu/drm/virtio/virtgpu_object.c
index 49fa59e09187b..1681486860019 100644
--- a/drivers/gpu/drm/virtio/virtgpu_object.c
+++ b/drivers/gpu/drm/virtio/virtgpu_object.c
@@ -159,9 +159,8 @@ static int virtio_gpu_object_shmem_init(struct virtio_gpu_device *vgdev,
 	shmem->pages = drm_gem_shmem_get_sg_table(&bo->base.base);
 	if (IS_ERR(shmem->pages)) {
 		drm_gem_shmem_unpin(&bo->base.base);
-		ret = PTR_ERR(shmem->pages);
 		shmem->pages = NULL;
-		return ret;
+		return PTR_ERR(shmem->pages);
 	}
 
 	if (use_dma_api) {
-- 
2.25.1

